src/client/components/LoginForm/Validation.js

```diff
@@ -0,0 +1,21 @@
+function Validation(values) {
+  const errors = {};
+  if (!values.email) {
+    errors.email = 'Email is required';
+  } else if (
+    !/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/.test(
+      values.email,
+    )
+  ) {
+    errors.email = 'Email address is invalid';
+  }
+
+  if (!values.password) {
+    errors.password = 'Password is required';
+  } else if (values.password.length < 6) {
```

Comment:
I recommended this pattern for password>"(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{6,}"
          "Must contain at least one number and one uppercase and lowercase letter, and at least 6 or more characters". I used same in signup form. so it will match.

Reply:
sure. will change it